Services/UI/OJS.Services.Ui.Models/Contests/ContestSortingServiceModel.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace OJS.Services.Ui.Models.Contests;
+
+public class ContestSortingServiceModel
+{
+    public string? SortType { get; set; }
```

Comment:
Shouldnt this be an enum also?

Reply:
I thought I deleted this file, but it is still there. Sort type enum property is in ContestFiltersServiceModel and this class is not needed and used.